build.gradle

```diff
@@ -4,6 +4,7 @@ plugins {
     id "java"
     id "edu.wpi.first.GradleRIO" version "2022.2.1"
     id "com.diffplug.spotless" version "6.2.0"
+    id "net.ltgt.errorprone" version "2.0.2"
```

Comment:
do we still need the errorprone compiler?

Reply:
Oops, you're right, we can drop this provided all the GuardedBy annotations are dropped